.github/workflows/sec3-test.yml

```diff
@@ -1,24 +1,24 @@
-name: Sec3
-on:
-  push:
-    branches: main
-  pull_request:
-    branches: "*"
-jobs:
-  audit:
-    runs-on: ubuntu-latest
-    timeout-minutes: 15
-    steps:
-      - name: Check-out the repository
-        uses: actions/checkout@v2
-      - name: Sec3 Pro Audit
-        continue-on-error: true    # set to true if you don't want to fail jobs
-        uses: sec3dev/pro-action@v1
-        with:
-          sec3-token: ${{ secrets.SEC3_TOKEN }}
-          path: hydra/program
-          hide-report-link: true
-      - name: Upload Sarif Report
-        uses: github/codeql-action/upload-sarif@v1
-        with:
-          sarif_file: sec3-report.sarif
+# name: Sec3
```

Comment:
Should probably put a comment with a reason for disabling and a note on how we could re-enable this properly.
Unless we totally gave up on this in which case we should just remove the file instead

Reply:
It fails in CI frequently and Austin said we can remove it so I'll just delete it. We have git history if we need to add it back later.